frontend/simple/components/CreateGroup/CreateGroupRules.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div id="rulesStep" class="has-text-centered">
+    <h1 class="title is-2"><i18n>Group Advanced Rules</i18n></h1>
+    <p class="content"><i18n>What percentage approval is necessary to adjust the group rules?</i18n></p>
+    <div class="columns">
+      <div class="column">
+        <p class="percent" v-if="!form.editChange">{{ group.changePercentage }}%</p>
+        <div class="field" v-if="form.editChange">
+          <div class="control">
+            <input
+              class="input is-large"
+              type="number"
+              min="1"
+              max="100"
+              name="changePercentage"
+              :value="group.changePercentage"
+              @input="update"
+            />
+          </div>
+        </div>
+        <p class="title is-5"><i18n>Change Rules</i18n></p>
+        <button id="changeRulesToggle" class="button is-link is-6 subtitle" @click="() => toggle('editChange')"><i18n>Change</i18n></button>
+      </div>
+      <div class="column">
+        <p class="percent" v-if="!form.editApprove">{{ group.memberApprovalPercentage }}%</p>
+        <div class="field" v-if="form.editApprove">
+          <div class="control">
+            <input
+              class="input is-large"
+              type="number"
+              min="1"
+              max="100"
+              name="memberApprovalPercentage"
+              :value="group.memberApprovalPercentage"
+              @input="update"
+            />
+          </div>
+        </div>
+        <p class="title is-5"><i18n>Add Member</i18n></p>
+        <button id="approveToggle" class="button is-link is-6 subtitle" @click="() => toggle('editApprove')"><i18n>Change</i18n></button>
+      </div>
+      <div class="column">
+        <p class="percent" v-if="!form.editRemove">{{ group.memberRemovalPercentage }}%</p>
+        <div class="field" v-if="form.editRemove">
```

Comment:
Two `v-if`'s next to each other like this, using the same variable, looks like an excellent candidate for moving line 43 below and putting in a `v-else` on it. :)